src/picterra/tracer_client.py

```diff
@@ -288,3 +292,162 @@ def list_plots_analyses(
         if page_number is not None:
             data["page_number"] = int(page_number)
         return self._return_results_page(f"plots_groups/{plots_group_id}/analysis/", data)
+
+    def list_plots_analysis_reports(
+        self,
+        plots_group_id: str,
+        plots_analysis_id: str,
+    ):
+        """
+        List all the reports belonging to a given plots analysis
+
+        Args:
+            plots_group_id: id of the plots group on which we want to list the analyses
+            plots_analysis_id: id of the plots analysis for which we want to list the reports
+
+        Returns: see https://app.picterra.ch/public/apidocs/plots_analysis/v1/#tag/reports/operation/getReportsList
+        """  # noqa[E501]
+        return self._return_results_page(
+            f"plots_groups/{plots_group_id}/analysis/{plots_analysis_id}/reports/"
+        )
+
+    def list_plots_analyses_report_types(
+        self,
+        plots_group_id: str,
+        plots_analysis_id: str
+    ):
+        """
+        List all the plots analyses report types the user can use (see create_plots_analysis_report)
+
+        Args:
+            plots_group_id: id of the plots group
+            plots_analysis_id: id of the plots analysis
+
+        Returns:
+            see https://app.picterra.ch/public/apidocs/plots_analysis/v1/#tag/reports/operation/getReportTypesForAnalysis
+        """  # noqa[E501]
+        resp = self.sess.get(
+            self._full_url(f"plots_groups/{plots_group_id}/analysis/{plots_analysis_id}/reports/types/")
+        )
+        _check_resp_is_ok(resp, "Couldn't list report types")
+        return resp.json()
+
+    def create_plots_analysis_report_precheck(
```

Comment:
Sorry to come on this late, but is this the right naming ? Like I get where it comes from, but reading it it feels like you're creating a precheck. Wondering if `run_create_plots_analysis_report_precheck` would be clearer.

I feel it's one we don't have to think about now and can observe a bit (in any case, easy to deprecate a method and alias it to another name later)